src/internal/kopia/merge_details_test.go

```diff
@@ -20,11 +32,33 @@ func TestDetailsMergeInfoerUnitSuite(t *testing.T) {
 	suite.Run(t, &DetailsMergeInfoerUnitSuite{Suite: tester.NewUnitSuite(t)})
 }
 
+func (suite *DetailsMergeInfoerUnitSuite) TestAdd_Twice_Fails() {
```

Comment:
Nit: in keeping with test naming standards.
```suggestion
func (suite *DetailsMergeInfoerUnitSuite) TestMergeDetails_AddRepoRef_duplicationFails() {
```